static/js/containers/CollectionDetailPage.js

```diff
@@ -188,6 +188,8 @@ export class CollectionDetailPage extends React.Component<*, void> {
 
     if (!collection) return null
 
+    document.title = `OVS | ${collection.title}`
```

Comment:
In open-discussions (and I think also in MM) we use a library called [react-document-title](https://github.com/gaearon/react-document-title), you may want to take a look at using that

Reply:
Thanks for the tip, done!